mx-semantics/main/accounts/configuration.md

```diff
@@ -1,31 +1,48 @@
 ```k
 
 module MX-ACCOUNTS-CONFIGURATION
-    imports INT-SYNTAX
-    imports STRING-SYNTAX
+    imports MX-ACCOUNTS-ADDRESS-CONFIGURATION
+    imports MX-ACCOUNTS-CODE-CONFIGURATION
+    imports MX-ACCOUNTS-ESDT-CONFIGURATION
     imports MX-STORAGE-CONFIGURATION
 
     configuration
         <mx-accounts>
             <mx-account multiplicity="*" type="Map">
-                // TODO: The address should be bytes.
-                <mx-account-address> "" </mx-account-address>
-                <mx-esdt-datas>
-                    <mx-esdt-data multiplicity="*" type="Map">
-                        // TODO: The esdt-id should be bytes.
-                        <mx-esdt-id>
-                            <mx-esdt-id-name> "" </mx-esdt-id-name>
-                            <mx-esdt-id-nonce> 0 </mx-esdt-id-nonce>
-                        </mx-esdt-id>
-                        <mx-esdt-balance> 0 </mx-esdt-balance>
-                    </mx-esdt-data>
-                </mx-esdt-datas>
+                <mx-account-address/>
+                <mx-esdt-datas/>
                 <mx-account-storage/>
+                <mx-account-code/>
             </mx-account>
         </mx-accounts>
 
 endmodule
 
+module MX-ACCOUNTS-ADDRESS-CONFIGURATION
+    imports STRING-SYNTAX
+
+    configuration
+        // TODO: The address should be bytes.
```

Comment:
Is any operation involving the account address or the `esdt-id` based on properties that come from them being bytes? If this is just for representation, having them as a string or as integers (like it is done in KEVM) can be simpler to handle.

Reply:
Normally, the address is a hash value, represented as a sequence of 32 bytes. For our example, I think it's fine to leave it as a String, since it's easier to use in various contexts (e.g. tests), it's more interesting to use descriptive values for addresses, and it should be easy to convert it to bytes whenever we decide to do that.